### ISSUE TITLE
every time keep-alive-timeout with Internet Explorer

### ISSUE THREAD

**Issue by bitfreak25** [eclipse/paho.mqtt.javascript#87]
I have tried different browser to test this library. As I used the Internet Explorer Version 11, I've got a disconnect after the keep-alive-timeout every time. It already works great with Firefox and Chrome, so I expect a problem with sending the keep-alive-message, only when using the Internet Explorer.

I've tested this with my own implementation and also with a local version of [this demonstration](https://eclipse.org/paho/clients/js/utility/).

**Comment by jpwsutton** [comment]
Hi,

I've booted up a VM, installed IE 11 on it and tested with a local copy of the  Utility with wireshark watching Websocket traffic and observed the following behaviour:
![screen shot 2016-10-20 at 10 02 31](https://cloud.githubusercontent.com/assets/1068763/19553903/1e9b47ae-96ae-11e6-9d8f-8fb9fcb5efd0.png)

At 23 seconds, Packet 21 is an ack coming back from the clients connect. After that there is a normal websocket PONG (Which doesn't factor into the keepalive). However at 83 seconds (Roughly 60 seconds later), (Packet 33) we see the Ping coming from the utility to the broker which responds correctly, this has continued for about 30 minutes now with no connection loss.

Could you confirm if you are using the master or develop branches, and what type / version of MQTT broker you are using. It would also be a good idea to test your code against iot.eclipse.org to make sure that your broker is not at fault.

**Comment by bitfreak25** [comment]
Hi,

I'm using mosquitto 1.4.10 from the Debian-Repository of mosquitto.org as the MQTT-Broker. 
My personal implementation of this demo is a snapshot I made many weeks ago. I replaced the JS-Libraries with local ones and changed some initial values for the connection. I also using the library in my own framework as a second implementation.

In my framework I get a Message, that the client is disconnected after approximately 60 seconds. With my own implementation of the Demo, when I wait approximately 60 seconds and click on "Disconnect" it reconnects instead of disconnecting. So I expect, that it did not show, that the client is disconnecting some time. I'm sorry for you to couldn't get more information, cause the developer-tools [F12] in IE are not installed for me.

Now I also tested a connection to iot.eclipse.org, as you preferred and it works. I also tested with the actual [Demonstration](https://eclipse.org/paho/clients/js/utility/) but the implementation in this link seems to be broken anyway. Because also a connection to iot.eclipse.org with Firefox did not work with this Demo. -> "SecurityError: The operation is insecure. (mqttws31.js:979:0)"
